libtcmu_log.c

```diff
@@ -455,7 +458,12 @@ static int create_file_output(struct log_buf *logbuf, int pri,
 	pthread_mutex_unlock(&logbuf->file_out_lock);
 	pthread_cleanup_pop(0);
 
+	tcmu_info("log file path now is '%s'\n", log_file_path);
 	return 0;
+
+err:
+	tcmu_dbg("Attempting to use '%s' as the log file path\n", log_file_path);
```

Comment:
I think I might have miswrote my review comment last time. I had meant that this debug statement should go before the open() call. If we put it in the error path I think it will be confusing because we would see something like:

Failed to open
Attempting to use '%s' as the log file path

which would be out of order.

Reply:
Yeah, make sense.